torchx/lib/torchx/backend.ex

```diff
@@ -674,8 +703,20 @@ defmodule Torchx.Backend do
 
   ## Ops
 
+  @impl true
+  def atan2(%{type: {:c, _}}, _l, _r) do
+    raise ArithmeticError, "Torchx does not support complex values for atan2"
+  end
+
+  def atan2(out, left, right) do
```

Comment:
I think you can skip this clause? And still keep it being autogenerated… or do you get a warning?

Reply:
Yes, I could skip it! I hadn't even tried because I though it would warn, but it seems that the compiler is nicer when it comes to autogenerated functions.